batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/controller/StudentControllerServlet.java

```diff
@@ -0,0 +1,179 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.services.StudentService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.swing.*;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by sanjay on 1/18/16.
+ */
+@WebServlet("/students/*")
+public class StudentControllerServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String pathInfo = request.getPathInfo();
+        String[] pathParts=null;
+        if(pathInfo !=null){
+            pathParts = pathInfo.split("/");
+            System.out.println("splitting the path");
+        }
+        else{
+            System.out.println("no the path to split null");
+        }
+       if(pathParts.length == 2 && pathParts[1].equals("create")){
+            createProcess(request, response);
+       }
+       else if(pathParts.length == 3 && pathParts[2].equals("delete"))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "delete" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)